### DISCUSSION TITLE
The WSJTX window

### DISCUSSION THREAD

**Discussion by rsaue** [foldynl/QLog#149] [Ideas]
Hi @foldynl,

I have a few suggestions on  filtering options on the WSJTX window, as there are no filtering options today

A screenshot of WSJTX window today: 

![wsjtx](https://user-images.githubusercontent.com/22942440/210107957-34d77db6-93de-4c21-aead-5e5b1b2397f4.png)

1.  Make a menu option to show only the last decode period. This will be nice on small monitors with little extra space. 

2. Make a menu option giving the user choice to filter out continent from view (EU/AS/NA/SA/AF/OC).

3. Make a menu option to Color Needed spots related to DXCC (new/new on band)

4. Make a menu option to Color spots QS0 B4 on band and/or to hide QSO B4 Calls on band

5. Make a menu option filtering all but coloured (needed) calls from view. 

73 de LB2EG Richard

**Comment by foldynl** [comment]
I'm just changing and testing a filter for DX Cluster window and noticed the same thing. I forgot to implement the filter in the WSJTX window. Thanks for the notice.
